content/_code-samples/use-tickets/py/use-tickets-multisig.py

```diff
@@ -0,0 +1,133 @@
+
+# Construct a TicketCreate transaction, 3 tickets will be created
+tx_create_ticket = TicketCreate(
+    account=myAddr,
+    ticket_count=3
+)
+
+# Sign transaction locally and submit
+tx_create_ticket_signed = safe_sign_and_submit_transaction(transaction=tx_create_ticket, wallet=test_wallet, client=client)
+
+# Get a Ticket Sequence
+get_ticket_sequence = client.request(AccountObjects(
+    account=myAddr,
+    type=AccountObjectType.TICKET
+))
+
+# Since we created 3 Tickets previously, you're able to choose which Ticket you're going to use
+ticket1_sequence = get_ticket_sequence.result["account_objects"][0]["TicketSequence"]
+ticket2_sequence = get_ticket_sequence.result["account_objects"][1]["TicketSequence"]
+ticket3_sequence = get_ticket_sequence.result["account_objects"][2]["TicketSequence"]
+
+print(f"\nTickets issued:\n"
+      f"Ticket 1: {ticket1_sequence}\n"
+      f"Ticket 2: {ticket2_sequence}\n"
+      f"Ticket 3: {ticket3_sequence}")
+
+ticket_sequence = int(input("\nPick 1 ticket sequence to use for your next multi-sig transaction: "))
+
+# Construct AccountSet Transaction using a Ticket
+tx_1 = AccountSet(
+    account=myAddr,
+    fee="1000",
+    sequence=0,
+    last_ledger_sequence=None,
+    ticket_sequence=ticket_sequence
+)
+
+autofilled_account_set_tx = autofill(tx_1, client, len(signers))
+
+# Sign 'tx_1' using all the keys on signers[]
+tx_result_1 = sign(transaction=autofilled_account_set_tx, wallet=signer_1_wallet, multisign=True)
+tx_result_2 = sign(transaction=autofilled_account_set_tx, wallet=signer_2_wallet, multisign=True)
+tx_result_3 = sign(transaction=autofilled_account_set_tx, wallet=signer_3_wallet, multisign=True)
+
+print(f"\nTx signature by account 1: {tx_result_1.signers}")
+print(f"Tx signature by account 2: {tx_result_2.signers}")
+print(f"Tx signature by account 3: {tx_result_3.signers}")
+
+# Combine all the signed transactions from the provided signed txs into a multi-sig transaction
+tx_1_filled = multisign(
+    transaction=autofilled_account_set_tx,
+    tx_list=[
+        tx_result_1,
+        tx_result_2,
+        tx_result_3
+    ]
+)
+
+response = client.request(SubmitMultisigned(tx_json=tx_1_filled))
+result = response.result
+
+print(f"\n Account Set tx result: {result['engine_result']}"
+      f"\n            Tx content: {result}\n")
+
+if result['engine_result'] == "tesSUCCESS":
+    print("Multi-sig transaction using a Ticket was successful!")
+elif result == "terPRE_TICKET":
+    print("The provided Ticket Sequence does not exist in the ledger")
+elif result == "tefMAX_LEDGER":
+    print("Transaction failed to achieve consensus")
+elif result == "tefPAST_SEQ":
+    print("The sequence number is lower than the current sequence number of the account")
+elif result == "unknown":
+    print("Transaction status unknown")
+else:
+    print(f"Transaction failed with code {result['engine_result']}")
```

Comment:
I'd recommend sharing the actual response code for all cases (that way people can look up what the error is if it fails, and can get used to seeing `tesSUCCESS` if it passes.
```suggestion
    print(f"The transaction's engine_result was {result['engine_result']}")
```